toxic_test.go

```diff
@@ -158,3 +158,91 @@ func TestFullstreamLatencyBiasUp(t *testing.T) {
 func TestFullstreamLatencyBiasDown(t *testing.T) {
 	DoLatencyTest(t, &LatencyToxic{Enabled: true, Latency: 100}, &LatencyToxic{Enabled: true, Latency: 1000})
 }
+
+func TestZeroLatency(t *testing.T) {
+	DoLatencyTest(t, &LatencyToxic{Enabled: true, Latency: 0}, &LatencyToxic{Enabled: true, Latency: 0})
+}
+
+func AssertEchoResponse(t *testing.T, client, server net.Conn) {
```

Comment:
We had something similar, but I had to rewrite it like this so we could test a single connection before and after the toxics were reset.